tests/e2e/outline-navigation.e2e.mjs

```
@@ -0,0 +1,28 @@
+import { browser } from '@wdio/globals';
+import { openWorkspaceFile, waitForCustomEditor, withCustomEditorWebview } from './helpers.mjs';
+
+describe('Outline navigation workflow', () => {
```

ESLint

[error] 'describe' is not defined.

(no-undef)